tools/buildsteps/windows/vswhere.bat

```diff
@@ -0,0 +1,63 @@
+@ECHO OFF
+
+
+REM Trick to make the path absolute
+PUSHD %~dp0\..\..\..\project\BuildDependencies
+SET builddeps=%CD%
+POPD
+
+SET arch=%1
+SET vcarch=amd64
+SET vcstore=%2
+SET vcvars=no
+SET sdkver=
+
+SET vsver=
+SET toolsdir=%arch%
+
+IF "%arch%" NEQ "x64" (
+  SET vcarch=%vcarch%_%arch%
```

Comment:
We should try with `amd64_%arch%` first and if that fails fall back to `%arch%` like we currently do it.

Reply:
do we really care about building on 32-bit versions?

Reply:
Will the `amd64_%arch%` version always be available if `%arch% is available?
If yes I don't care much, otherwise it has to be adjusted.

Reply:
Should be. There's no separate option to install one or the other
![image](https://user-images.githubusercontent.com/874927/34469302-4b6cbd36-ef1c-11e7-8319-98b9ddd01207.png)